noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/TagTweetGeotagNotRequired.scala

```diff
@@ -0,0 +1,24 @@
+package edu.uci.ics.cloudberry.noah.feed
+
+import java.io.File
+
+import edu.uci.ics.cloudberry.gnosis._
+import edu.uci.ics.cloudberry.noah.adm.TweetGeotagNotRequired
+import edu.uci.ics.cloudberry.util.Profile._
+import twitter4j.{TwitterException, TwitterObjectFactory}
+
+object TagTweetGeotagNotRequired {
```

Comment:
Same comments as the `TweetGeotagNotRequired`. It doesn't seem worth to create a `class` or `object`. Instead, we should extend the existing function to add the `throw or not throw` option.